pong-game/backend/app/chat/routing.py

```diff
@@ -2,7 +2,6 @@
 from . import consumers
 
 websocket_urlpatterns = [
-    # re_path(r"ws/chat-server/", consumers.ChatConsumer.as_asgi()),
-    re_path(r"^ws/chat-server/(?P<room_name>\w+)/$", consumers.ChatConsumer.as_asgi()),
+	re_path(r"ws/chat-server/(?P<room_name>\w+)/$", consumers.ChatConsumer.as_asgi()),
```

Comment:
I wonder why you remove the anchor `^` for the ChatConsuer?

The `^` anchor restricts that the URI must START with `ws/chat-server/room_name/` which is a more exact method. So if there uri that looks like `/any_potential_prefix/ws/chat-server/room/` won't work.

Without `^`, as long as part of the URI matches, it will still count. If there are other concerns that we might need it to be more flexible, then I'm okay.